core/src/test/java/forklift/consumer/injection/ServiceNamedBeanResolver.java

```diff
@@ -0,0 +1,23 @@
+package forklift.consumer.injection;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import forklift.decorators.BeanResolver;
+import forklift.decorators.Service;
+
+@Service
+public class ServiceNamedBeanResolver {
+    static Map<String, Person> map = new HashMap<>();
+
+    @BeanResolver
+    public Object resolve(Class<?> requestedClass, String name){
```

Comment:
Does scope matter at all?  It looks like if we have a named bean then we are applying `Singleton` scope and if we have a non-named bean we are creating new instances on every request

Reply:
This is fine, since it it only being used in the tests to prove that the named annotation resolver can be used successfully. This is not production level code.